CMakeLists.txt

```diff
@@ -15,9 +15,11 @@ endif()
 
 set(CMAKE_INSTALL_DOCDIR "" CACHE PATH "documentation root (DATAROOTDIR/doc/${PROJECT_NAME}${LIBIIO_VERSION_MAJOR}-doc)")
 include(GNUInstallDirs)
-if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-	set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_FULL_LIBDIR}")
-	set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+if(ENABLE_SHARED)
+	if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
```

Comment:
You can make the patch smaller by just writing ```if(ENABLE_SHARED AND ${CMAKE_SYSTEM_NAME} MATCHES "Linux")```

Reply:
Indeed, you're right.
Just updated, thanks !